src/core/render/jsx/render.unit.tsx

```diff
@@ -232,7 +235,7 @@ describe('render', () => {
 
   describe('<Host>', () => {
     it('should merge component host with <Host>', async () => {
-      const fixture = new ElementFixture();
+      const fixture = new ElementFixture(TEST_CONFIG);
```

Comment:
Not related to this line, but

Line 269 has an unused component. Can we delete it?